### DISCUSSION TITLE
How to fix "errCode": "UNABLE_TO_VERIFY_LEAF_SIGNATURE" (artifactory)

### DISCUSSION THREAD

**Discussion by Stefano05Farris** [renovatebot/renovate#19378] [Request Help] [answered]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

34.55.0

### If you're self-hosting Renovate, select which platform you are using.

Azure DevOps Server

### If you're self-hosting Renovate, tell us what version of the platform you run.

Version Azure DevOps Server 2020 Update 1.2

### Was this something which used to work for you, and then stopped?

I never saw this working

### Problem
we are currently in the process of adding conan support for our client project. The conan packages updates should be searched on our private artifactory.

We are able to authenticate renovatebot on the artifactory so it can search for conan dependency updates in the private artifactory, but we encounter this error message: `UNABLE_TO_VERIFY_LEAF_SIGNATURE`.

Testing with `process.env['NODE_TLS_REJECT_UNAUTHORIZED'] = '0';` added to the config.js gives us the desired result, however, this is not a safe option.

When checking the certificate authority used from the specified url renovatebot searches for dependency updates it indicates that it is trusted. We also looked at nodejs's certificate authority list where it is also trusted.

### Relevant debug logs

<details><summary>Logs</summary>

```
         "datasource": "conan",
         "packageName": "PACKAGENAME/1.44.0@_/_",
         "url": undefined,
         "errCode": "UNABLE_TO_VERIFY_LEAF_SIGNATURE"

```

</details>

### Have you created a minimal reproduction repository?

No reproduction repository

**Comment by rarkins** [comment, marked as answer]
https://docs.renovatebot.com/examples/self-hosting/#self-signed-tlsssl-certificates

Please create Discussions and not Issues in future.
